downloader/src/main/kotlin/vcs/GitRepo.kt

```diff
@@ -60,22 +62,29 @@ private data class Include(
     val name: String
 )
 
-class GitRepo : GitBase() {
+class GitRepo : VersionControlSystem(), CommandLineTool {
     override val type = VcsType.GIT_REPO
     override val priority = 50
+    override val defaultBranchName = "master"
+    override val latestRevisionNames = listOf("HEAD", "@")
+
+    override fun command(workingDir: File?) = "repo"
+
+    override fun getVersion() =
+        throw IOException("The $type version can only be determined from an initialized working tree.")
```

Comment:
Consider writing just 'repo' of '$type'.

Reply:
Done, plus single-quoting the command name.